content/en/continuous_integration/quality_gates/_index.md

```diff
@@ -81,6 +81,7 @@ The behavior of the command can be modified using the following flags:
 based on the current pipeline context. By default, the command succeeds.
 - `--fail-if-unavailable`: when this flag is specified, the command fails if one or more rules could not be evaluated because of an internal issue.
 By default, the command succeeds.
+- `--timeout`: the command will stop its execution after the specified timeout in seconds. The default timeout is 10 minutes.
```

Comment:
Can you describe cases where it can happen?

Reply:
I've updated the description, wdyt?